dotnet/targets/Microsoft.MaciOS.Sdk.Xcode.targets

```diff
@@ -0,0 +1,205 @@
+    </XcodeBuild>
+
+    <!-- Create XCFRAMEWORK for $(TargetPlatformIdentifier) -->
+    <!-- TODO Sign the XCFramework bundle? Looks like this is done via _CodesignAppBundle but may need to sign nested content? -->
+    <ItemGroup>
+      <_XcArchiveOutputs Include="%(XcodeProject.OutputPath)archives/%(XcodeProject.SchemeName)$(TargetPlatformIdentifier).xcarchive" />
+      <_XcArchiveOutputs Include="%(XcodeProject.OutputPath)archives/%(XcodeProject.SchemeName)$(TargetPlatformIdentifier)Simulator.xcarchive"
+          Condition=" Exists('%(XcodeProject.OutputPath)archives/%(XcodeProject.SchemeName)$(TargetPlatformIdentifier)Simulator.xcarchive') "/>
+    </ItemGroup>
+    <XcodeBuild
+        Command="-create-xcframework"
+        FrameworkArchives="@(_XcArchiveOutputs)"
+        FrameworkName="%(XcodeProject.SchemeName).framework"
+        FrameworkOutputPath="%(XcodeProject.OutputPath)xcframeworks/%(XcodeProject.SchemeName)$(TargetPlatformIdentifier).xcframework"
+        WorkingDirectory="%(XcodeProject.RootDir)%(XcodeProject.Directory)" >
+    </XcodeBuild>
+
+    <ItemGroup>
+      <_XcodeProjectXcFrameworkOutputs Include="%(XcodeProject.OutputPath)xcframeworks/%(XcodeProject.SchemeName)$(TargetPlatformIdentifier).xcframework"
+          Condition=" '%(CreateNativeReference)' == 'true' "
+          Kind="%(Kind)"
+          SmartLink="%(SmartLink)"
+          Visible="%(Visible)" />
+    </ItemGroup>
+
+    <Message Text="Adding reference to Xcode project output: '%(_XcodeProjectXcFrameworkOutputs.Identity)'. The '%25(CreateNativeReference)' metadata can be set to 'false' to opt out of this behavior." Importance="high" />
+
+    <Touch Files="$(_BuildXcodeProjectsStamp)" AlwaysCreate="true" />
+    <ItemGroup>
+      <!-- Add stamp and build outputs to FileWrites so they are not deleted on IncrementalClean -->
+      <FileWrites Include="$(_BuildXcodeProjectsStamp);%(XcodeProject.OutputPath)**/*" />
+      <!-- Automatically add XCFRAMEWORK outputs to be bound or otherwise processed later in the build -->
+      <NativeReference Include="@(_XcodeProjectXcFrameworkOutputs)" Kind="%(Kind)" SmartLink="%(SmartLink)" Visible="%(Visible)" />
+    </ItemGroup>
+  </Target>
+
+
+  <Target Name="_GetSharpieBindInputs">
+    <ItemGroup>
+      <_SharpieInputs Include="%(XcodeProject.OutputPath)archives/%(XcodeProject.SchemeName)$(TargetPlatformIdentifier).xcarchive" />
+    </ItemGroup>
+  </Target>
+
+  <!-- TODO put this in a task? -->
+  <Target Name="_GetSharpieBindSdkName"
+      Condition=" '@(XcodeProject->Count())' != '0' and '@(XcodeProject->'%(SharpieBind)')' == 'true' and '$(SharpieBindSdkName)' == '' " >
+    <Exec Command="xcode-select -p"
+        Condition=" $([MSBuild]::IsOSPlatform('osx')) "
+        ConsoleToMSBuild="true"
+        StandardOutputImportance="low">
+      <Output TaskParameter="ConsoleOutput" PropertyName="_XcodeSelectOutput" />
+    </Exec>
```

Comment:
If you depend on the `_DetectSdkLocations` task, you'll find the Xcode location in the `_SdkDevPath` property afterwards, there's no need to do `xcode-select -p`.

Reply:
Thanks I've removed this target, but am now using _SdkDevPath elsewhere as needed.